src/main/java/hudson/model/RadiatorView.java

```diff
@@ -102,11 +126,26 @@
 	 *            Caption text to be used on the radiator's headline.
 	 * @param captionSize
 	 *            Caption size for the radiator's headline.
+	 * @param useBackgrounds
+	 *            if background images for status display should be used
+	 * @param otherImage
+	 *            URL for not built jobs
+	 * @param successImage
+	 *            URL for successful jobs
+	 * @param unstableImage
+	 *            URL for unstable jobs
+	 * @param brokenImage
+	 *            URL for failed jobs
 	 */
 	@DataBoundConstructor
 	public RadiatorView(String name, Boolean showStable,
 			Boolean showStableDetail, Boolean highVis, Boolean groupByPrefix,
-			Boolean showBuildStability, String captionText, Integer captionSize) {
+			Boolean showBuildStability, String captionText, Integer captionSize,
+			Boolean useBackgrounds,
+			String otherImage,
+			String successImage,
+			String unstableImage,
+			String brokenImage) {
```

Comment:
I am sorry, I am not very familiar with how Jenkins proposes to do this and just followed along the already existing pattern.